src/models/event-timeline-set.ts

```diff
@@ -159,17 +158,12 @@ export class EventTimelineSet extends EventEmitter {
      *
      * @throws If <code>opts.pendingEventOrdering</code> was not 'detached'
      */
-    public getPendingEvents(thread?: Thread): MatrixEvent[] {
+    public getPendingEvents(): MatrixEvent[] {
         if (!this.room || !this.displayPendingEvents) {
             return [];
         }
 
-        const pendingEvents = this.room.getPendingEvents(thread);
-        if (this.filter) {
-            return this.filter.filterRoomTimeline(pendingEvents);
-        } else {
```

Comment:
@germain-gg I'm trying to understand this change. I note it changed the behaviour of this method so that it no longer applies the TimelineSet's filter (despite what the documentation says), but  really my question is: why doesn't the calling code just call `room.getPendingEvents`, if it actually wants all the pending events rather than going through the filter?